NotifyMe/utils/exceptionManager.py

```diff
@@ -0,0 +1,68 @@
+
+# proper response format-> { status code, a message, an actual data, status code, 
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+
+class CustomException(Exception): # CUSTOM EXCEPTION
+  def __init__(self, message, data, status_code):
```

Comment:
this constructor should take error_code, message, exception parameter  and error_code should be predefined code e.g. USER_NOT_FOUND=1001, message should be user understandable format and e should be exception object